site/content/en/docs/concepts/resource_flavor.md

```diff
@@ -3,25 +3,44 @@ title: "Resource Flavor"
 date: 2022-03-14
 weight: 2
 description: >
-  An object that you can define to describe what resources are available
-  in a cluster.
+  An object that defines available compute resources in a cluster and enables fine-grained resource management by associating workloads with specific node types.
 ---
 
 Resources in a cluster are typically not homogeneous. Resources could differ in:
 
-- pricing and availability (for example, spot versus on-demand VMs)
-- architecture (for example, x86 versus ARM CPUs)
-- brands and models (for example, Radeon 7000 versus Nvidia A100 versus T4 GPUs)
+- Pricing and availability (for example, spot versus on-demand VMs)
+- Architecture (for example, x86 versus ARM CPUs)
+- Brands and models (for example, Radeon 7000 versus Nvidia A100 versus T4 GPUs)
 
-A ResourceFlavor is an object that represents these resource variations and
-allows you to associate them with cluster nodes through labels, taints and tolerations.
+A ResourceFlavor is an object that represents these resource variations and allows you to associate them with cluster nodes through labels, taints and tolerations.
 
 {{% alert title="Note" color="primary" %}}
-If the resources in your cluster are homogeneous, you can use 
-an [empty ResourceFlavor](#empty-resourceflavor) instead of adding labels to custom ResourceFlavors.
+If the resources in your cluster are homogeneous, you can use an [empty ResourceFlavor](#empty-resourceflavor) instead of adding labels to custom ResourceFlavors.
 {{% /alert %}}
 
-A sample ResourceFlavor looks like the following:
+## ResourceFlavor tolerations for automatic scheduling
+
+**Requires Kubernetes 1.23 or newer**
+
+This approach may be best for teams that wish to schedule pods onto the appropriate nodes automatically. However, a limitation arises when multiple types of specialized hardware are present, such as two different nvidia.com/gpu resources present in the cluster, i.e., T4 and A100 GPUs. The system may not differentiate between them, meaning the pods could be scheduled on any of both types of hardware.
```

Comment:
TBH I tend to ask people to use new lines for each sentence as this gets hard to read in markdown. 

```suggestion
This approach may be best for teams that wish to schedule pods onto the appropriate nodes automatically.
However, a limitation arises when multiple types of specialized hardware are present, such as two different nvidia.com/gpu resources present in the cluster, i.e., T4 and A100 GPUs. 
The system may not differentiate between them, meaning the pods could be scheduled on any of both types of hardware.
```

Reply:
Yes, I understand your point with Markdown. These docs will ultimately be read from the website which makes sense to have them in a single parragraph. - I could do a 'break' on each sentence, while keeping it in the same parragraph if that suits? - Please see latest commit as an example

Reply:
Yea that is great.